src/Caliburn.Micro.Avalonia/Platforms/netcore-avalonia/BootstrapperBase.cs

```diff
@@ -102,13 +101,12 @@ protected virtual void StartRuntime()
         /// </summary>
         protected virtual void PrepareApplication()
         {
-            //TODO: (Avalonia) Implement application events when Avalonia supports them
-            
-            //Application.Startup += OnStartup;
-
-            //Aplication.DispatcherUnhandledException += OnUnhandledException;
```

Comment:
Avalonia indeed does not seem to have something like this, nor is it planned. But I think adding a try/catch in Program.cs is close enough.

Reply:
ok